wiki/Article_Styling_Criteria/en.md

```diff
@@ -82,150 +100,232 @@ tags:
+
+Do:
+**You are not allowed to post R-18 skins on the osu! forums.**
+```
+
+#### Bold
+
+Use bold when you need to caution or note something's importance. When bolding, try to avoid bolding single word(s) but instead bold phrases, as it might be hard to notice the bolding on a single word. Prefer double asterisks (`**`) over double underscores (`__`).
+
+#### Italics
+
+Use italics only when naming a video game or the name of a work. osu!, the game, is exempt from this. Prefer single asterisks (`*`) over single underscores (`_`).
+
+### Headings
+
+Limit to a heading level of 5 and never use headings to style or format text. Prefer the [ATX (hash) style](https://github.github.com/gfm/#atx-headings) over the [Setext (underline) style](https://github.github.com/gfm/#setext-headings).
+
+Optionally, add an extra line before and after the heading to help make the heading stand out while editing.
+
+#### Titles
+
+All articles must have one level 1 heading, being the article's title. Article titles must use title case and never contain formatting, links, nor images. The title of an article is the name of the folder that the article is located in.
+
+The level 1 heading should be placed at the start of the article unless you have reference links or metadata placed there. If this is the case, place the level 1 heading underneath it.
+
+For English article titles, if you need to reword the title, you must rename the folder to match the article title. There are rare exceptions to this however.
+
+#### Sections
+
+All section headings are to be using levels 2 to 5. Section headings are to use sentence case and should never contain styles nor links. In addition to this, section headings must never skip a heading level (e.g. do not go from a level 2 heading to a level 4 heading). The first section heading must be a level 2 heading. Unlike titles, section headings can have small image icons.
+
+In the osu! wiki, heading levels 4 and 5 are hidden in the table of contents.
 
 ### Lists
 
-There are two kinds of lists: bulleted and numbered. The spacing for both types of bullets is very important! Please limit to 4 indentations/sub-bullets.
+There are two kinds of lists: bulleted and numbered. The spacing for both types of bullets is very important! Limit to 4 indentations/sub-bullets.
 
 #### Bulleted
 
-Use bulleted lists when the order of the list does not matter. It is preferred to use the hyphen (`-`) or plus symbol (`+`) instead of the asterisk (`*`) because the single asterisk is already used for italics.
+Use bulleted lists when the order of the list does not matter. It is preferred to use the hyphen (`-`) or the plus symbol (`+`) instead of the asterisk (`*`) because the single asterisk is already used for italics.
 
 #### Numbered
 
-Use numbered lists when the order of the list does matter. Incrementing the numbers is optional.
+Use numbered lists when the order of the list does matter. Do not increment the numbers. For example:
```

Comment:
Why do you not increment the numbers? How about steps like in the [Installation page](https://osu.ppy.sh/help/wiki/Installation)?

Reply:
> Why do you not increment the numbers?

Markdown recognizes that this:
```
1. First
2. Second
3. Third
```
1. First
2. Second
3. Third

is the same as this:
```
1. First
1. Second
1. Third
```
1. First
1. Second
1. Third

I'm half and half about this, but I thought that if we increment the numbers, we'll need to fix the proceeding numbers if, for example, a random step in between 6 and 7 was added to a 15 step guide.

Reply:
So, this is code thing then, oops. Thought it was a preview thing. Should add something about `code`.

Reply:
Is the example given good enough?
![image](https://user-images.githubusercontent.com/14264992/40954513-87a1bcbe-684a-11e8-8eaa-ecaad172cfaf.png)

Reply:
It’s fine. Was just confused that you can do this in the code and it’ll preview it with increments.

Reply:
Alright, I just did some more testing and realized that parsedown/osu-web does set a start attribute for numbered lists. Will remove.